tripleblindmarket/covid-safe-paths
# Load raw html eulas, and upload changes to Lokalise automatically

Uses raw html EULA files so that they can be pushed to Lokalise automatically

EULA text was not changed, however it was tidied and formatted with Prettier. I did a manual eye check and it looks to have zero changes.

Also:

- Fixed the EULA close button alignment
- Adds a `yarn:i18n push` which will upload all english files to Lokalise.
- Added `en.html` to the push.

#### Linked issues:



#### Screenshots:

<img width="317" alt="Screen Shot 2020-04-28 at 1 58 50 PM" src="https://user-images.githubusercontent.com/702990/80537420-e6c4b580-8958-11ea-9e61-8d590ce8896d.png">
<img width="487" alt="Screen Shot 2020-04-28 at 2 00 09 PM" src="https://user-images.githubusercontent.com/702990/80537427-e9270f80-8958-11ea-828f-1f01f9ff9549.png">


#### How to test

Test the EULA loads for English, Haitian and that English EULA loads for any other language.

Check that the EULA modal can be continued.. (because the scroll check was removed)
